test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -530,7 +530,10 @@ private void mkdirs()
     private Config loadConfig(IInstanceConfig overrides)
     {
         Map<String,Object> params = ((InstanceConfig) overrides).getParams();
-        return YamlConfigurationLoader.fromMap(params, Config.class);
+        boolean check = true;
+        if (overrides.get("dtest.api.config.check") != null)
```

Comment:
@dcapwell I suppose it shouldn't change, but creating a constant for this that's also used in `newInstanceWrapper()` might be a good idea.

Reply:
changed, but also moved the cluster_id one (same pattern) to a new Constants class.